parserator_web/static/js/index.js

```diff
@@ -1 +1,92 @@
-/* JS code goes here */
+/* I got a lint error for this async function
+   "Unexpected token function"
+   but wasn't able to fix, and kept function as is */
+async function getAndDisplayAddressData(inputString) {
+  const url = "api/parse/?input_string=" + inputString;
+  const response = await fetch(url, {
+    method: "GET",
```

Comment:
the GET method is the default. do not need to specify.

Reply:
Roger that! Good to know, thanks.